lms/static/js/spec/components/header/header_spec.js

```diff
@@ -0,0 +1,51 @@
+(function (define) {
+    'use strict';
+
+    define(['jquery',
+            'underscore',
+            'js/components/header/views/header',
+            'js/components/header/models/header'
+           ],
+           function($, _, HeaderView, HeaderModel) {
+
+               describe('header component view', function () {
+                   var model, view;
+
+                   beforeEach(function () {
+                       model = new HeaderModel({
+                           title: 'Test title',
+                           description: 'Test description'
+                       });
+                       view = new HeaderView({
+                           model: model
+                       });
+                   });
+
+                   it('can render itself', function () {
+                       expect(view.$el.text()).toContain('Test title');
+                       expect(view.$el.text()).toContain('Test description');
+                   });
+
+                   it('does not show breadcrumbs by default', function () {
+                       expect(view.$el.html()).not.toContain('<nav class="breadcrumbs">');
+                   });
+
+                   it('shows breadcrumbs if they are supplied', function () {
```

Comment:
It would be good to have tests with a single breadcrumb as well as with two. If you factor out a helper method it should be straightforward to reuse most of the logic.